src/components/controls/button.scss

```diff
@@ -31,4 +31,22 @@
 
 .secondary:hover {
     background-color: darken(#fff, 10);
-}
\ No newline at end of file
+}
+
+.round {
+    background-color: $busBlue;
+    color: #fff;
+    padding: 10px 15px 10px 15px;
+    border-radius: 20px;
+    width:100%;
+    white-space: nowrap;
+    margin-right: 15px;
```

Comment:
Move to .button padding, white-space nowrap, margin-write

move width: 100% to .form.scss .button { width: 100% }
(reason: buttons should work the same way by default, this round should only make button's borders round)